viewer/package.json

```diff
@@ -76,7 +76,6 @@
   },
   "dependencies": {
     "@cognite/potree-core": "^1.1.3",
-    "@cognite/sdk": "^2.28.0",
```

Comment:
@larsmoa we can try to add them as dev dependencies to avoid build failures

```
npm WARN @cognite/reveal@1.0.0-alpha-1 requires a peer of @cognite/sdk@^2.28.0 but none is installed. You must install peer dependencies yourself.
npm WARN @cognite/reveal@1.0.0-alpha-1 requires a peer of three@^0.115.0 but none is installed. You must install peer dependencies yourself.
```